ts/features/pn/aar/components/errors/__tests__/SendAARErrorComponent.test.tsx

```diff
@@ -117,6 +161,90 @@ describe("SendAARErrorComponent - Full Test Suite", () => {
     const { toJSON } = renderComponent();
     expect(toJSON()).toMatchSnapshot();
   });
+
+  it("zendeskAssistanceLogAndStart calls expected functions in order", () => {
```

Comment:
This test is not testing the real code. We are testing that the code that is written in the test is executed in the proper order. Instead, we should:
- mock the return value of `currentAARFlowStateAssistanceErrorCode`
- spy on each methods from `utils/supportAssistance`. You can do it like:
```
import * as supportAssistance from "../../../../../../utils/supportAssistance"
const spiedOnMockedAppendLog = jest.spyOn(supportAssistance, "appendLog").mockImplementation();
````
- render `SendAARErrorComponent`
- fire a press for the `button_assistance` button
- check that the spied assistance methods have been called (proper number of times, proper parameters) and in the proper order